### DISCUSSION TITLE
몇가지 기능상 중요한 질문을 하고싶습니다.(기능, UI)

### DISCUSSION THREAD

**Discussion by smartbh** [naverz/zepeto-studio-kor#16] [5. Q&A]
1. 

지금 현재 개발시 기본 UI는

![image](https://user-images.githubusercontent.com/43138813/144073471-3e6e4e9f-f5bc-4001-8082-0d008729abff.png)

이런식인것으로 알고 있습니다.

그런데 제가 저 상자와 같은 앉을수 있는 오브젝트에 갔을때

다른 공식 제페토 월드들 처럼

![image](https://user-images.githubusercontent.com/43138813/144073593-dc91233e-0971-4c46-8b63-c5dc6f682175.png)

이런식으로 상호작용 버튼이 나오게 하고 싶은데

예제를 찾기가 너무 어렵습니다.

특정 사물에 가서 UI버튼이 화면에 뜨게 하고

그 버튼에 특정 행동이나 제스처를 취할 수 있게 하는 기능을 알고싶습니다.

2.

1번 질문과 연관된 연장 질문인데

제페토 월드 중 스키점프 맵을 보면

월드에서 자유롭게 돌아다니다가

특정 플레이 위치만 가면 플레이를 할 수가

있는거로 알고있습니다. 하지만 스키점프는 혼자서 플레이 하는 형식인데

다같이 기다렸다가 최대 4명이 차면 시작을 하는 형식으로 만들고 싶습니다.

방장 같은 형식은 따로 없구요

그런식의 UI기능을 넣고싶은데 혹시 좋은 예제가 있을지 궁금합니다.

**Comment by NoahHahm** [comment]
1. 아직 상호작용에 대한 피쳐는 개방되어 있지 않습니다. 직접 구현하셔야 합니다.
2. 멀티플레이 서버에서 OnJoin 함수 호출되는 시점 (사용자들이 접속되면 호출되는 시점)에 ClinetCount 를 확인하고, 4명이 되면 클라이언트로 이벤트를 보내서 처리하는 방식을 취해야 합니다. 간단한 샘플은 아직 제공되고 있지 않지만 내부에 전달해보겠습니다.
